pages/common/enca.md

```diff
@@ -1,20 +1,20 @@
 # enca
 
-> Detect and convert encoding of text files.
+> Detect and convert the encoding of text files.
 > More information: <https://github.com/nijel/enca>.
 
-- Detect file(s) encoding according to your system's locale:
+- Detect file(s) encoding according to the system's locale:
 
-`enca {{file(s)}}`
+`enca {{file1 file2 ...}}`
 
-- Detect file(s) encoding; -L option tells enca the current language; language is in the POSIX/C locale format, e.g. zh_CN, en_US etc:
+- Detect file(s) encoding specifying a language the POSIX/C locale format (e.g. zh_CN, en_US):
```

Comment:
``` suggestion
Detect file(s) encoding specifying a language in the POSIX/C locale format (e.g. zh_CN, en_US):
```

Reply:
Looks  like the preceding dash got lost there, @andrik

Reply:
Fixed, thanks.